### PR TITLE
[WFLY-18185] Community documentation for caching realm allowing authentication with externally updated credential

### PR DESCRIPTION
**Pull request by skyllarr**
https://issues.redhat.com/browse/WFLY-18185

Requires: https://github.com/wildfly-security/wildfly-elytron/pull/1932

### CODE REVIEW

File: docs/src/main/asciidoc/_elytron/migrate/Caching_Migration.adoc [hunk 1]
```diff
@@ -83,7 +83,7 @@ These can then be used in a security domain and subsequently an authentication f
 ./subsystem=elytron/http-authentication-factory=application-security-http:add(http-server-mechanism-factory=global, security-domain=application-security, mechanism-configurations=[{mechanism-name=BASIC}])
 ----
 
-In this final step it is very important that the caching-realm is referenced rather than the original realm otherwise caching will be bypassed.
+In this final step it is very important that the caching-realm is referenced rather than the original realm otherwise caching will be bypassed. When credentials are updated externally in the LDAP server, the obsolete user's credential will be removed when the user successfully authenticates with the updated credential.
```
Comment (fjuma) [inline docs/src/main/asciidoc/_elytron/migrate/Caching_Migration.adoc:86]: s/the obsolete user's credential/the user's obsolete credential

Comment (fjuma) [reply docs/src/main/asciidoc/_elytron/migrate/Caching_Migration.adoc:86]: @Skyllarr I think this change still needs to be made.

**Review by fjuma — CHANGES_REQUESTED** [review]
Just one small update needed for this one. Otherwise looks good. Thanks @Skyllarr!

File: docs/src/main/asciidoc/_elytron/migrate/Caching_Migration.adoc (hunk 1 above)
Comment (Skyllarr) [reply docs/src/main/asciidoc/_elytron/migrate/Caching_Migration.adoc]: @fjuma Thank you, fixed!

**Review by fjuma — APPROVED** [review]
Thanks @Skyllarr!

**Review by darranl — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @Skyllarr, @fjuma and @darranl

**Pull request merged**